toolchain/check/testdata/interface/no_prelude/fail_member_lookup.carbon

```diff
@@ -45,7 +45,10 @@ fn F() {
 
 interface Different {}
 
-// CHECK:STDERR: fail_member_lookup.carbon:[[@LINE+7]]:24: error: cannot implicitly convert value of type `Different` to `Interface` [ImplicitAsConversionFailure]
+// CHECK:STDERR: fail_member_lookup.carbon:[[@LINE+10]]:24: error: cannot implicitly convert value of type `Different` to `Interface` [ImplicitAsConversionFailure]
+// CHECK:STDERR: fn G(U:! Different) -> U.(Interface.T);
+// CHECK:STDERR:                        ^~~~~~~~~~~~~~~
+// CHECK:STDERR: fail_member_lookup.carbon:[[@LINE+7]]:24: note: expected type implementing `Interface`; found non-type value [ConversionFailureNonTypeToFacetNote]
```

Comment:
This is another case that seems misleading to say "non-type value" to describe `U` which is a facet.